config/initializers/countries.rb

```diff
@@ -0,0 +1,10 @@
+ISO3166::Data.register(
```

Comment:
Yes. Kosovo won't be in the list otherwise. The countries gem doesn't include it, because not all countries recognize it.